DEPS

```diff
@@ -11,17 +11,25 @@ Core dependencies
 
 Essential:
 
-* python (== 2.7.x)
+* python3 (>= 3.6)
 * bsddb (distributed with stdlib) or bsddb3
```

Comment:
bsddb was removed from the stdlib, so this now requires the external bsddb3 module.

Reply:
Thanks!